packages/experimental/Avatar/src/suffixes.ts

```diff
@@ -0,0 +1,107 @@
+export const suffixes = new Set<string>([
```

Comment:
Going to be a really nit picky here and say technically these are Titles. "Suffix" would only apply to the items that would go after a name, but a lot of these would go before the name.

Also the list seems to be missing "Sr"

Reply:
I agree. I also wanted to name it titles, but I got this name form the Design spec. I'll rename